1.2/spec.md

```diff
@@ -551,73 +551,76 @@ Something that requires JPEG or PNG to be represented well
 Inline Representations {#inline-representation}
 ======================
 
+Issue(51):
+
 There is some content that should behave and flow like text
 
-Classes for Inline Representation {#inline-classes}
+Unrecognized characters and words {#unrecognized}
 ---------------------------------
 
-### <dfn element>ocr_glyph</dfn>
-
-An individual glyph represented as an image (e.g., an unrecognized character)
-
-Must contain a single `<img>` tag, or be present on one
-
-### <dfn element>ocr_glyphs</dfn>
-
-Multiple glyphs represented as an image (e.g., an unrecognized word)
-
-Must contain a single `<img>` tag, or be present on one
-
-### <dfn element>ocr_dropcap</dfn>
+  : <dfn element>ocr_glyph</dfn>
+  :: An individual glyph represented as an image (e.g., an unrecognized character)
+  :: Must contain a single <{img}> tag, or be present on one
 
-An individual glyph representing a dropcap
+  : <dfn element>ocr_glyphs</dfn>
+  :: Multiple glyphs represented as an image (e.g., an unrecognized word)
+  :: Must contain a single <{img}> tag, or be present on one
 
-May contain text or an `<img>` tag; the `alt` of the image tag should contain
-the corresponding text
-
-### <dfn element>ocr_chem</dfn>
-
-A chemical formula
-
-Must contain either a single `<img>` tag or [[CML]] markup, or be present on
-one
-
-### <dfn element>ocr_math</dfn>
+Dropcap {#dropcap}
+-------
 
-A mathematical formula
+  : <dfn element>ocr_dropcap</dfn>
+  :: An individual glyph representing a dropcap
+  :: May contain text or an <{img}> tag; the `alt=` of the image tag should contain
+    the corresponding text
 
-Must contain either a single `<img>` tag or [[MathML]] markup, or be present on
-one
+Mathematical and chemical formulas {#formulas}
+----------------------------------
 
 Mathematical and chemical formulas that float must be put into an <{ocr_float}>
 section.
 
 Mathematical and chemical formulas that are “display” mode should be put into
 an <{ocr_display}> section.
 
-### Superscript and Subscript
+: <dfn element>ocr_chem</dfn>
+:: A chemical formula
+:: Must contain either a single <{img}> tag or [[CML]] markup, or be present on one
+
+: <dfn element>ocr_math</dfn>
+:: A mathematical formula
+:: Must contain either a single <{img}> tag or [[MathML]] markup, or be present on one
+
+Superscript and Subscript {#sub-sup}
+-------------------------
 
 Superscripts and subscripts, when not in <{ocr_math}> or <{ocr_chem}> formulas,
-must be represented using the HTML `<sup>` and `<sub>` tags, even if special
+must be represented using the HTML <{sup}> and <{sub}> tags, even if special
```

Comment:
Fixed, thanks https://www.w3.org/TR/html5/text-level-semantics.html#the-sub-and-sup-elements